### DISCUSSION TITLE
Next big feature poll #2

### DISCUSSION THREAD

**Discussion by seerge** [seerge/g-helper#525] [Polls]
Hello, just wanted to run a poll to gather opinions of what next big feature would be most helpful. 

**Top 2 winners were implemented : Driver update checker / built in debloat.**

**Comment by schm1dtmac** [comment]
The ability to start/stop/disable all ASUS services from G-Helper directly, on startup especially, could be a good way of ensuring that they never manage to re-enable/interfere with the system after major Windows Updates/Feature Updates that upgrade the OS itself.

Such upgrades appear to re-enable all System Control Interface services that may have been manually disabled prior, so they would need to be re-disabled manually by the user, but if G-Helper were to handle disabling them directly then I guess it would 'bypass' the problem per-se.

**Reply by seerge** [reply]
Yeah, that's what _"Button to stop or start all asus services (i.e. built in debloat.bat)"_ is about :)

**Reply by seerge** [reply]
But I think i first need to do driver updates in app, so people won't depend on myasus anymore, then I can safely offer disabling all asus services

**Reply by Metalogic999** [reply]
A driver update checker in G-Helper would be great thanks! Then hopefully after that a direct way to stop/start Asus Services which are not necessary when running G-Helper.

**Reply by seerge** [reply]
Checker is live in 0.79

**Comment by bowes332frank** [comment]
Thanks for your APP

**Comment by xschwi** [comment]
Driver update checker would be cool. In addition, basic driver installation assistant after clean windows installation might be even better :D

**Comment by vocaloidict** [comment]
I'd like to be able to bind screen brightness up/down to M3 and M4. I tried to see if this is possible by using virtual key codes but I can't find any information on either the Fn key, or how to do key combinations (i.e. if I can do Fn + F7/F8 then that would solve things)

**Reply by seerge** [reply]
Try this one [GHelper.zip](https://github.com/seerge/g-helper/files/11677888/GHelper.zip)

**Comment by seerge** [comment]
Added Start / Stop all asus services button (under Extra)

**Reply by Metalogic999** [reply]
Does that stop/start the same Asus services as the debloat.bat/bloat.bat scripts? Thanks

**Reply by seerge** [reply]
Yes, it's just integrated debloat/bloat bat

**Reply by schm1dtmac** [reply]
Would it be possible to allow for auto-stopping/disabling services whenever G-Helper is launched (e.g. as an extra option)? Usually the button being clicked once would be enough to stop ASUS services from coming back until the next Windows feature update, but having it auto-stop/disable them each time would help to ensure that, on each feature update, no user interaction is needed to kill the bloat services. (Also helps for Insiders in the Dev/Canary channels as we get flighted builds each week or so typically)

**Reply by seerge** [reply]
That's an overkill, as asus services will get "re-installed" only when new Asus Control Interface updates (which happens like once in a half of the year).+ I want to keep app as unintrusive (by default) as possible
